jps/jupedsim/IO/worldparser.py

```diff
@@ -0,0 +1,137 @@
+import re
+from typing import List
+
+import ezdxf
+import jpscore
+
+
+class WorldParser:
+    """
+    Class for parsing dxf files of the world and creating corresponding geometry objects.
+    """
+
+    def __init__(self, p_input_file: str):
+        """
+        Initialization with a dxf file. In case of valid file format Lines and SpecialAreas are parsed.
+        :param p_input_file: name of the dxf file
+        """
+        self.m_input_file = p_input_file
+
+        # try to open file
+        try:
+            doc = ezdxf.readfile(self.m_input_file)
+        except IOError:
+            # TODO throw exception. file not found
+            print("File not found")
+            return
+        except ezdxf.DXFStructureError:
+            # TODO throw exception. invalid dxf file.
+            print("Invalid dxf file format")
+            return
+
+        # create modelspace
+        self.m_msp = doc.modelspace()
+
+        # read in meta data
+        self.m_unit = doc.header["$INSUNITS"]
+        # allow only 6 for meters, TODO: check diff to LUNITS
+        if self.m_unit != 6:
+            # TODO accept other units as well (but not all)
+            # TODO throw exception
+            print("Length unit is not meter. Not yet implemented")
+
+        # TODO check $MEASUREMENT = 0 English not metric
+        # NOTE: these headers are not available in qcad file
+        # upper_right_corner = doc.header['$EXTMAX']
+        # lower_left_corner = doc.header['$EXTMIN']
+
+        # build World
+        self.m_jps_world_builder = jpscore.geometry.WorldBuilder()
+        self.__parseLevels()
+
+    @staticmethod
+    def parseCoordinates(line: str) -> List[float]:
+        """
+        Method parses a dxf line with coordinates to a list
+        :param line: start and end coordinate of a dxf line formatted as string '(start_x, start_y, start_z) (end_x, end_x, end_z)'
+        :return: coordinates as a list [start_x, start_y, end_x, end_y]
```

Comment:
Maybe something like: 
```suggestion
        :return: coordinates as a list [{"x": x, "y": y}, {"x": x, "y": y}]
```
But i am not sure if this any better.

Reply:
The function returns a List of geometry.Coordinates now